src/osd/osd_types.cc

```diff
@@ -5878,3 +5878,33 @@ ostream& operator<<(ostream& out, const store_statfs_t &s)
       << ")";
   return out;
 }
+
+void OSDOp::clear_data(vector<OSDOp>& ops)
+{
+  for (unsigned i = 0; i < ops.size(); i++) {
```

Comment:
@tchaikov
One of the reason I decide to use static function is seeing the functions defined above clear_data()
ex: static void merge_osd_op_vector_out_data(vector<OSDOp>& ops, bufferlist& out);
(in osd_types.h)

The use cases are very similar.